zetasql/fuzzing/simple_evaluator_fuzzer.cc

```diff
@@ -0,0 +1,26 @@
+//
+// Copyright 2019 ZetaSQL Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include <stdint.h>
+#include <stddef.h>
```

Comment:
Are these frist two include needed? If so, use the C++ versions.

```suggestion
#include <cstdint>
#include <cstddef>
```

Reply:
I think the problem is that the the following code will be linked to C code so c style header is preferred? Also, uint8_t is used as opposed to std::uint8_t (which is the declared function signature by libFuzzer) in the function header.